1-exercises/A-this/exercise-3.js

```diff
@@ -8,8 +8,12 @@
 
 let bankAccount = {
   currentAmount: 100,
-  deposit: function (amount) {},
-  withdraw: function (amount) {},
+  deposit: function (amount) {
```

Comment:
Hi @Alirezabg,
I'm just doing the code review hw and I have to leave a line suggestion. The methods look good to me, but if you are feeling lazy feel free to try the shorthand format, so for e.g. if you want to you could write this:
```
let bankAccount  = {
    deposit: function (amount) { ... },
    withdraw: function (amount) { ...}
}

like this:

let bankAccount = {
    deposit (amount) { ... },
    withdraw (amount) { ... }
}
```
See this page if you are interested: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions 😃